src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -349,13 +347,17 @@ class ContentSidebar extends PureComponent<Props, State> {
      */
     fetchVersionsErrorCallback = (e: Error) => {
         this.setState({
-            versions: undefined,
+            versions: {
+                total_count: 0,
+                entries: []
+            },
             versionError: {
```

Comment:
Can we remove this?

Reply:
Yup planning on removing it in a separate commit. It's being passed through to the DetailsSidebar right now but not being used anywhere